client/app/components/ServicesTemplate/data.json

```diff
@@ -0,0 +1,53 @@
+{
+  "data":
+    {
+      "topCardContent":
+        [{
+          "id": "1",
+          "title": "Name and Address",
+          "content": "Verify telephone numbers, accounts, email addresses, names, SSN, and addresses",
+          "url": "telified",
+          "img": "3D"
```

Comment:
![screen shot 2018-01-05 at 8 31 41 am](https://user-images.githubusercontent.com/10067791/34618212-e3b4ef86-f1f2-11e7-8850-ac5516d28f59.png)

Images not provided, I assume we need to get fresh svgs from @leseanbiz or can we reuse existing ones for this section?

Reply:
i will try and get svgs ASAP for this and the other services below.